README.md

```diff
@@ -1,23 +1,26 @@
-Operator Framework Community
-============================
+# Operator Framework Community
 
 This repository contains community process, charter, and coordination documents for the Operator Framework Project,
 
 Designs, technical proposals, and issues are tracked in the [operator-sdk repo](https://github.com/operator-framework/operator-sdk/tree/master/doc) and [operator-lifeycle-manager repo](https://github.com/operator-framework/operator-lifecycle-manager/tree/master/doc/design).
 
-Get involved
-------------
+## Get involved
 
 Discuss on:
 * [Kubernetes Slack in #kubernetes-operators](https://kubernetes.slack.com) ([Get invite](https://slack.k8s.io))
 * [Operator Framework on Google Groups](https://groups.google.com/forum/#!forum/operator-framework)
 * [Operator Framework Commons SIG](https://commons.openshift.org/sig/OpenshiftOperators.html)
 
-Operator Framework Special Interest Group Meetings
------------------------------------------
+## Meetings
+
+### Operator Framework Special Interest Group
 
 * Next Scheduled Meeting: November 12 2019  16:00 UTC/9:00 am Pacfic/Noon Eastern [dial-in details](https://github.com/operator-framework/community/projects/2#card-27939134)
-* [Proposed Agenda](https://github.com/operator-framework/community/projects)
-* [Propose an agenda topic propose](https://github.com/operator-framework/community/projects/2#column-6841651)
-  
+  * [Proposed Agenda](https://github.com/operator-framework/community/projects)
+  * [Propose an agenda topic propose](https://github.com/operator-framework/community/projects/2#column-6841651)
   
+### Operator SDK 
+
```

Comment:
Could not be possible it be and google calendar event? I mean, would be great if we could click on it and add in our google calendar. Maybe the agenda could be there too.

Reply:
As far as I know, the way this is typically handled in the Kubernetes community is that one of the maintainers sends the invite to the mailing list. That way, everyone on the mailing list gets it without having to find this page.

See https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit

I could be wrong, but it seems that embedding a calendar in this way would require that we have a public calendar? And we would probably some sort of governance around that? Not sure I want to go down that rabbit hole, at least not as a blocker to getting this meeting on the books.